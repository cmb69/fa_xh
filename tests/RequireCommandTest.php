<?php

namespace Fa;

use ApprovalTests\Approvals;
use PHPUnit\Framework\TestCase;

class RequireCommandTest extends TestCase
{
    /** @var array<string,string> */
    private array $conf;

    protected function setUp(): void
    {
        $this->conf = XH_includeVar("./config/config.php", "plugin_cf")["fa"];
    }

    private function sut(): RequireCommand
    {
        return new RequireCommand("./", $this->conf);
    }

    public function testWritesToHjs(): void
    {
        global $hjs;
        $hjs = "";
        $this->sut()->execute();
        Approvals::verifyHtml($hjs);
    }
}
