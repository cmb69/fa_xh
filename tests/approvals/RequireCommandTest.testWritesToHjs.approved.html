<link rel="stylesheet" type="text/css" href="./css/font-awesome.min.css">